src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -2377,4 +2377,110 @@ def test_containerapp_max_inactive_revisions(self, resource_group):
         self.cmd(f'containerapp update -g {resource_group} -n {app} --cpu 0.25 --memory 0.5Gi')
         self.cmd(f'containerapp show -g {resource_group} -n {app}', checks=[JMESPathCheck("properties.configuration.maxInactiveRevisions", maxInactiveRevisions)])
 
-        
\ No newline at end of file
+
+class ContainerappRuntimeTests(ScenarioTest):
+    def __init__(self, *arg, **kwargs):
+        super().__init__(*arg, random_config_dir=True, **kwargs)
+
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="northcentralus")
+    def test_containerapp_runtime_java_metrics_create(self, resource_group):
+        TEST_LOCATION = "northcentralusstage"
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+
+        app = self.create_random_name(prefix='aca', length=24)
+        image = "mcr.microsoft.com/k8se/quickstart:latest"
```

Comment:
Do we have any image that can test the java runtime? The quickstart image seems can always be deployed successfully.

Reply:
I asked this question is we should use a more suitable image to test the support scenarios.